profile/README.md

```diff
@@ -0,0 +1,11 @@
+# Welcome to Pulsar
+
+Pulsar, or sometimes referred to as Pulsar-Edit is a new image of the beloved 'Hackable Text Editor' [Atom](https://github.com/atom/atom).
+
+After the announcement of [Atom's sunset](https://github.blog/2022-06-08-sunsetting-atom/) the community came together to keep Atom alive, over on a longstanding fork [Atom-Community](https://github.com/atom-community/atom).
```

Comment:
After the announcement of [Atom's sunset](https://github.blog/2022-06-08-sunsetting-atom/), the community came together to keep Atom alive via the longstanding fork [Atom-Community](https://github.com/atom-community/atom).

Reply:
You should be able to request changes directly on the code, which allows proper attribution. Otherwise these all look good, and I'd be happy to make them. Just let me know if you'd like to do so. I'm trying to find the documentation on that process

Reply:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request